package/MDAnalysis/coordinates/GRO.py

```diff
@@ -140,7 +140,7 @@ def _read_first_frame(self):
             for pos, line in enumerate(grofile, start=-2):
                 # 2 header lines, 1 box line at end
                 if pos == n_atoms:
-                    unitcell = np.array(map(float, line.split()))
+                    unitcell = np.array(list(map(float, line.split())))
```

Comment:
a better function here could be `np.float32(line.split())`. It does the array conversion automatically.